invenio_drafts_resources/services/records/tasks.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Celery tasks to manage drafts."""
+
+from datetime import timedelta
+
+from celery import shared_task
+from invenio_base.utils import obj_or_import_string
+
+
+@shared_task(ignore_result=True)
+def cleanup_drafts(current_service_imp, seconds=3600):
+    """Hard delete of soft deleted drafts.
+
+    :param int seconds: numbers of seconds that should pass since the
```

Comment:
doubt: is it since the last update? or since it was "deleted" (soft)... because a draft that is being updated (i.e. in progress but not published yet) is not mark as deleted (which the query of the api.py is doing).

Reply:
I believe that deleting updated the timestamp of the draft_model.updated property.